packages/core/phase2/lib/getOperationSequence/areAllResultsAlreadyPresentOnPnc/isMatchToPncAdjAndDis/isPncDisposalMatch.ts

```diff
@@ -1,12 +1,14 @@
 import type { PncDisposal } from "../../../../../types/PncQueryResult"
 
+const areStringsEqual = (obj1: string | undefined, obj2: string | undefined) => (!obj1 && !obj2) || obj1 === obj2
```

Comment:
`null`, `undefined`, and empty string are all considered equivalent in this context.